src/util/units.spec.js

```diff
@@ -16,20 +16,67 @@
 
 'use strict';
 
-import { fromWei } from './units';
+import '@polkadot/types/injector';
+
+import extrinsicsFromMeta from '@polkadot/api-metadata/extrinsics/fromMetadata';
+import { Balance, Compact, createType, GenericCall, Metadata } from '@polkadot/types';
+import Call from '@polkadot/types/primitive/Generic/Call';
+import kusamaData from './static-kusama';
+import { formatDecimal, fromWei } from './units';
 
 describe('units', () => {
-  it('should properly convert units from wei', () => {
-    let wei = '5208';
-    let ether = fromWei(wei);
-    expect(ether).toEqual('0.000000000000021');
+  describe('ethereum', () => {
+    it('should properly convert units from wei', () => {
+      let wei = '5208';
+      let ether = fromWei(wei);
+      expect(ether).toEqual('0.000000000000021');
+    });
+  
+    it('should return BigNumber for undefined values', () => {
+      expect(fromWei(null)).toEqual('0');
+      expect(fromWei(undefined)).toEqual('0');
+      expect(fromWei(0)).toEqual('0');
+      expect(fromWei('0')).toEqual('0');
+      expect(fromWei('')).toEqual('0');
+    });
   });
 
-  it('should return BigNumber for undefined values', () => {
-    expect(fromWei(null)).toEqual('0');
-    expect(fromWei(undefined)).toEqual('0');
-    expect(fromWei(0)).toEqual('0');
-    expect(fromWei('0')).toEqual('0');
-    expect(fromWei('')).toEqual('0');
+  describe('kusama', () => {
+    beforeAll(() => {
+      const metadata = new Metadata(kusamaData);
+    
```

Comment:
```suggestion
```

Reply:
ok I will just use `formatBalance` and do the same thing apps does then. The only thing is the "international" friendly space instead of commas, which I can make a PR to add that option for in the `common` library, but I don't think there's any international standard for that, so no matter which we choose, some people will be less than pleased

Reply:
Sure, what I meant by international is that we shouldn't use comma as a thousand separator, anything else would be good.

Reply:
cool, thankfully Jaco has already thought of this in his formatBalance method :)